### PR TITLE
Detaching Classic wallet

### PR DESCRIPTION
**Pull request by deleterium**
- [x] Update selector page
- [x] Create task on release workflow
- [x] Delete Classic wallet files
- [x] Test release workflow

### CODE REVIEW

File: ci/updateClassic.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+TMPDIR=$(mktemp -d)
+cd $TMPDIR > /dev/null
+
+# download
+echo
+echo "======================================="
+echo "⬇️ Cloning Classic wallet main branch..."
+echo "---------------------------------------"
+git clone https://github.com/signum-network/signum-classic-wallet.git
+# gh repo clone signum-network/signum-classic-wallet
```
Comment (ohager) [inline ci/updateClassic.sh:20]: could be removed then

File: html/ui/classic/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Signum Classic Wallet moved to another repository!
```
Comment (ohager) [inline html/ui/classic/README.md:1]: :+1:

**Review by ohager — APPROVED** [review]
just a minor cleanup required, but rest LGTM

**Comment by deleterium** [comment]
I've created a fork, draft releases and made soem tests:
- [x] Classic Wallet
- [x] Phoenix Wallet
- [x] api-doc (openapi)
- [x] api-doc (legacy)
- [x] node connect and one transaction made.

**Comment by frankTheTank72** [comment]
@jjos2372 can we add this to the next relase on testnet pls

**Pull request merged**